<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstStartClass</class>
 <widget class="QDialog" name="FirstStartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>572</width>
    <height>339</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WinMusik 3 - First Start</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/logo/resources/icon64.png</normaloff>:/logo/resources/icon64.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>166</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/other/resources/setup-wizzard.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>391</width>
     <height>277</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>This is the first time you started WinMusik. The setup assistant will help you to configure the application after your requirements. All preferences can be changed afterwards within the setup menu.

Please click &quot;next&quot; to continue.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome to WinMusik 3!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New installation or recovery?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>If WinMusik 3.x was installed previously on your computer, you can use the old database. If you want to do that, please select the database directory of the old installation. All options and data will be reused.

If this should be a new installation, simply click on &quot;next&quot;.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayout_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>381</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recoverpath">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectrecoverpath">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons/resources/folder-open-16.png</normaloff>:/icons/resources/folder-open-16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database directory</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please select the directory in which WinMusik should save it's database.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayout_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>381</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="localdatapath">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectdatapath">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons/resources/folder-open-16.png</normaloff>:/icons/resources/folder-open-16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The configuration parameters are complete now. You can change a lot more options in the options dialog after starting WinMusik.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Configuration completed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Click on &quot;finish&quot; to save the configuration parameters and start WinMusik.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The configuration parameters of the former WinMusik installation will be used. Click on &quot;finish&quot; to start WinMusik now.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Configuration completed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayout">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QPushButton" name="buttonZurueck">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/resources/previous.png</normaloff>:/icons/resources/previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonWeiter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>next</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/resources/next.png</normaloff>:/icons/resources/next.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonAbbrechen">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/icons/resources/cancel.png</normaloff>:/icons/resources/cancel.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources.qrc"/>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
